iuriMorari/src/main/java/AdminUser.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Map;
+
+public class AdminUser extends AbstractUser {
+
+    public AdminUser(String username, Map<String, Address> addresses) {
+        super(username, addresses);
+    }
+
+    public AdminUser(String username) {
+        super(username);
+    }
+
+    public void approveOrder(AbstractOrder userOrder) {
+        System.out.println("Order " + userOrder.anyID() + " is approved");
```

Comment:
in such case you will print out an unknown ID, make shure that you are going to print out the manipulated order ID

Reply:
not sure if understood correctly, but done